yaksh/static/yaksh/js/requesthandler.js

```diff
@@ -53,16 +53,32 @@ function response_handler(method_type, content_type, data, uid){
     } else if(content_type.indexOf("application/json") !== -1) {
         res = JSON.parse(data);
         request_status = res.status;
-        if(method_type === "POST") {
-            uid = res.uid;
+        if (request_status){
+          if(method_type === "POST") {
+              uid = res.uid;
+          }
+          check_state(request_status, uid);
+        }
+        else{
+          unlock_screen();
+          var error_output = document.getElementById("error_panel");
+          error_output.innerHTML = res.error;
+          focus_on_error();
         }
-        check_state(request_status, uid);
     } else {
         reset_values();
         unlock_screen();
     }
 }
 
+function focus_on_error(){
+    var ele = document.getElementById('error_panel')
```

Comment:
Just a suggestion you don't need to get element again you can pass the element as an argument to this function from response_handler.